docs/source/en/training.mdx

```diff
@@ -63,12 +63,18 @@ If you like, you can create a smaller subset of the full dataset to fine-tune on
 
 <a id='trainer'></a>
 
-## Train
-
 <frameworkcontent>
 <pt>
 <Youtube id="nvBXf7s7vTI"/>
 
+## Train
+
+At this point, you should follow the section corresponding to the framework you want to use. You can use the links
+in the right sidebar to jump to the one you want - and if you want to hide all of the content for a given framework,
+just use the button at the top-right of that framework's block!
+
```

Comment:
Same for this intro.

Reply:
Love this little intro for guiding user navigation/improving readability!